src/test/resources/Features/surveillanceComplaints.feature

```diff
@@ -72,3 +73,18 @@ Scenario: Associate a criteria to a complaint
     And I click save button
     And I click Edit complaint for ONC-ACB Complaint ID "ONCACB005"
     Then the criteria "170.315 (a)(1)" should be associated to the complaint
+    
+Scenario: Validate compliant status based on closed date
```

Comment:
'complaint'- I don't want to hold off the PR for typos but it'd be nice to fix those if you can in this one